### DISCUSSION TITLE
Docker with Morphir

### DISCUSSION THREAD

**Discussion by DamianReeves** [finos/morphir#125] [General]
I think Docker might be a good way to deliver some of our tooling to our users.
I know some Docker work has been done, but do we have the `morphia-elm` tools available in a Docker image at this point?

/cc @AttilaMihaly , @stephengoldbaum

**Comment by stephengoldbaum** [comment]
We have containerized the Develop App.  That allows teams to easily boot up a visualizer for serving transparency.  We should look at extending that.  How do you envision using Docker for delivering other tooling?

**Reply by DamianReeves** [reply]
> We have containerized the Develop App. That allows teams to easily boot up a visualizer for serving transparency. 

Is it deployed to Docker Hub?

**Reply by AttilaMihaly** [reply]
The Develop App runs on an IR that is included in the image so we could only deploy one to Docker Hub with a specific IR. We chose to implement it as a CLI command instead that you can run on a model to create an image from.

**Reply by stephengoldbaum** [reply]
Other than the Develop App, how would Docker be useful?

**Reply by AttilaMihaly** [reply]
Unfortunately it's not included in our main docs yet but here's a pointer: https://github.com/finos/morphir-elm/blob/73f207929ea1161cd12dff6789250013b18c137d/cli2/morphir-dockerize.ts#L12
